plugins/allowtagsfrompaste/trumbowyg.allowtagsfrompaste.js

```diff
@@ -133,7 +133,7 @@
 						setTimeout(function () {
 							var processNodes = trumbowyg.$ed.html();
 							$.each(trumbowyg.o.plugins.allowTagsFromPaste, function(iterator, tagName) {
-								processNodes = processNodes.replace(new RegExp("</?" + tagName + "[^>]*>", "gi"), "");
+								processNodes = processNodes.replace(new RegExp("<\\/?" + tagName + "(\\s[^>]*|)>", "gi"), "");
```

Comment:
`"<\\/?" + tagName + "(\\s[^>]*|)>"`

Should be: 

`"<\\/?" + tagName + "(\\s[^>]*)?>"`

No?

Reply:
No, it looks funny, but it either ends with space + params or nothing at all:
<span foo=“bar”>
</span>

Reply:
Ah actually `"<\\/?" + tagName + "(\\s[^>]*)?>` works as well indeed, 
guess it doesn't really matter